src/rulesets/__tests__/merger.unit.ts

```diff
@@ -58,7 +59,7 @@ describe('Rulesets merger', () => {
       },
     });
 
-    expect(ruleset).toHaveProperty('rules.test.severity', 'off');
+    expect(ruleset).toHaveProperty('rules.test.severity', -1);
```

Comment:
Should we just add `-1` to `DiagnosticSeverity` for consistency?

Reply:
I considered doing that, but AFAIR our initial intention was to follow LSP and I believe they don't have such a level, would need to check.